nav2_smart_planner/src/smart_planner.cpp

```diff
@@ -269,6 +269,25 @@ SmartPlanner::makePlan(
   return !plan.poses.empty();
 }
 
+void
+SmartPlanner::smoothApproachToGoal(
```

Comment:
I think a comment explaining what this function is doing would be helpful.

Reply:
Will add comments.